docs/use-cases/payments/index.page.tsx

```diff
@@ -0,0 +1,318 @@
+      description: "EUROP, issued by Schuman Financial, is the first MiCA-compliant euro stablecoin on XRPL.",
+    },
+    {
+      id: "xsgd",
+      label: "XSGD",
+      url: "https://www.straitsx.com/xsgd",
+      description: "XSGD, from StraitsX, is a Singapore Dollar-backed stablecoin regulated by MAS.",
+    },
+    {
+      id: "audd",
+      label: "AUDD", 
+      url: "https://www.audd.digital/",
+      description: "AUDD, an Australian dollar stablecoin, is live on XRPL and backed 1:1 with AUD.",
+    },
+  ];
+
+  const embeddedPaymentsCards = [
+    {
+      id: 'digital-wallets',
+      title: 'Digital Wallets',
+      description: 'Offer fast, low-fee stablecoin payments between users and applications.',
+    },
+    {
+      id: 'cross-border-remittance',
+      title: 'Cross-Border Remittance',
+      description: 'Use secure payment channels and the most optimal liquidity pathways for global remittances with RLUSD.',
+    },
+    {
+      id: 'regulated-foreign-exchange',
+      title: 'Regulated Foreign Exchange',
+      description: 'Tap into a set of fiat-backed stablecoins, instantaneous swaps for efficient Foreign Exchange.',
+    },
+    {
+      id: 'merchant-settlement',
+      title: 'Merchant Settlement',
+      description: 'Settle daily payments across assets using escrow or checks with compliance-focused features.',
+    },
+    {
+      id: 'b2b-payment-rails',
+      title: 'B2B Payment Rails',
+      description: 'Build programmable payment flows with conditions and real-time data feeds.',
+    },
+    {
+      id: 'compliance-first-payment-acceptance',
+      title: 'Compliance-First Payment Acceptance',
+      description: 'Add Deposit Authorization and whitelisting to comply with AML and KYC workflows.',
+    },
+  ];
+
+  const battleTestedProjects = [
+    {
+      id: "coinpayments",
+      label: "CoinPayments",
+      url: "https://xrpl.org/blog/2025/coinpayments-xrpl-case-study-payment-processing",
+      description: "CoinPayments uses XRPL's fast and low-cost payment rails to enable merchants to accept digital assets globally with near-instant settlement and minimal transaction fees.",
+      buttonText: "Case Study"
+    },
+    {
+      id: "ripple",
+      label: "Ripple",
+      url: "https://ripple.com/solutions/cross-border-payments/",
+      description: "Ripple Payments enables crypto companies, payment service providers and fintech to facilitate real-time cross-border payments using stablecoins, digital assets and local currencies — with XRPL as a foundational transaction layer.",
+      buttonText: "Case Study"
+    },
+    {
+      id: "fiipay",
+      label: "FiiPay",
+      url: "https://xrpl.org/blog/2025/frii-pay-xrpl-case-study-crypto-payment-solution",
+      description: "FiiPay connects XRPL-based crypto wallets to point-of-sale terminals, allowing customers to pay with RLUSD or XRP while helping merchants save costs on card processing fees.",
```

Comment:
It's supposed to be "FriiPay" (as in, sounds like "free", not like "fee", I think)

```suggestion
      id: "friipay",
      label: "FriiPay",
      url: "https://xrpl.org/blog/2025/frii-pay-xrpl-case-study-crypto-payment-solution",
      description: "FriiPay connects XRPL-based crypto wallets to point-of-sale terminals, allowing customers to pay with RLUSD or XRP while helping merchants save costs on card processing fees.",
```